SQL/Cleanup_patches/2018-11-20-remove_currentGUITable_field.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE parameter_type DROP COLUMN CurrentGUITable;
```

Comment:
@driusan @xlecours any idea if this is still doing anything ??

Reply:
no clue

Reply:
it's not